plugin/restore/mysql/mysql_test.go

```diff
@@ -397,67 +397,92 @@ func TestGetReplayBinlogPathList(t *testing.T) {
 func TestParseAndSortBinlogFileName(t *testing.T) {
 	a := require.New(t)
 	tests := []struct {
-		binlogFileNames []string
-		expect          []binlogItem
+		binlogFileNames []BinlogFile
+		expect          []BinlogFile
 		err             bool
 	}{
 		// Normal
 		{
-			binlogFileNames: []string{"binlog.000001", "binlog.000002", "binlog.000003"},
-			expect: []binlogItem{
+			binlogFileNames: []BinlogFile{{Name: "binlog.000001"}, {Name: "binlog.000002"}, {Name: "binlog.000003"}},
```

Comment:
Add a specific case for 999999 and above since that's mentioned in the comment?

Reply:
Added.